### ISSUE TITLE
Topic: SDPi-xC with Mixed Device Safety Classes

### ISSUE THREAD

**Issue by ToddCooper** [IHE/sdpi-fhir#61]
See [Topic: SDPi-xC with Mixed Device Safety Classes on confluence](https://confluence.hl7.org/x/ThEvBQ). 

NOTE:  This topic will remain open after SDPi 1.0 **BUT** it was determined to add "guidance" for the issue in 1.0.  

Specifically ([from 12 Aug, item 3.b](https://confluence.hl7.org/x/qhsLC)):  For SDPi 1.0, metrics and alerts can be kept simple with guidance for when to use Inf(ormation) & MedA, with MedB & MedC out-of-scope for SDPi 1.0   Also, it should be determined where in TF-1, -2 & -3 this guidance should be included ... possibly all three.  For example, a general discussion in the SDPi-P, -R & -A profile (Overview / Concepts) sections, perhaps a TF-2  if there is technical / transactional relevance, and definitely in TF-3 for the BICEPS semantics handling in general for MDIB elements that have a SafetyClassification associated with them.  Also factor in the SES sections - perhaps in the boilerplate template for those sections.

- [ ] Determine the editorial approach (near and long term) for integration of BICEPS SafetyClassification in to SDPi specifications
- [ ] Add SafetyClassification content into the SDPi specification
- [ ] Add the Mixed Safety Classes interim (SDPi 1.0) guidance into the SDPi specification

**Comment by ToddCooper** [comment]
Update related [sdpi-supplement-issues.adoc](https://github.com/IHE/sdpi-fhir/blob/master/SDPi_Supplement/asciidoc/sdpi-supplement-issues.adoc)  item when complete, indicating where the updates were made or the branche(s) that captured the updates, as well as moving from OPEN to CLOSED lists if appropriate.

**Comment by MartinKasp** [comment]
**Text proposal:** 

The purpose of the mandatory attribute @SafetyClassification is to classify the criticality of information including external control operation. The usage of this attribute is discussed for the different purposes. Additionally, the core definitions of the value set are under discussion. Consequently, this document does not include deeper information about the usage of @SafetyClassification in this version but will be updated later.

**Comment by JavierEspina** [comment]
> **Text proposal:**
> 
> The purpose of the mandatory attribute @SafetyClassification is to classify the criticality of information including external control operation. The usage of this attribute is discussed for the different purposes. Additionally, the core definitions of the value set are under discussion. Consequently, this document does not include deeper information about the usage of @SafetyClassification in this version but will be updated later.

The (good) text proposed above seems focused on external control. It talks about "external control operation" + claims that @SafetyClassification is manadatory, which AFAIK is only true for external control [in the glue std.] and will probably be the case in the future for alerts [early A-PKP draft std.]. If similar text is needed for more general parts (e.g. SES sections of SDPi-P and SDPI-R) I propose this variation:

_The purpose of the BICEPS attribute @SafetyClassification is to classify the quality and criticality of data and operations. The usage of this attribute is under discussion in IEEE 11073 PoCD and IHE DEV DPI. Additionally, the value set definition in BICEPS is under discussion. Consequently, this document version does not yet include guidance or requirements on the usage of @SafetyClassification._

**Comment by MartinKasp** [comment]
Yes, you are right: glue:R0028 make the attribute mandatory. Due to the implied value "Inf", the attribute is mandatory to consider the attribute as the provider would provide wrong information if the attribute is omitted and does not fit to the category "Inf".

**Comment by ToddCooper** [comment]
For SDPi 1.0, a Reviewers Note box was added to the BICEPS SES Requirements and Considerations section.  This should suffice until the issue is fully resolved by the IEEE PoCD SDC working group.
